client/components/data/purchases/site-purchases/index.jsx

```diff
@@ -1,65 +0,0 @@
-/**
- * External dependencies
- */
-import React from 'react';
-
-/**
- * Internal dependencies
- */
-import { fetchSitePurchases } from 'lib/upgrades/actions';
```

Comment:
We should remove this `fetchSitePurchases` action - it doesn't look like it's used anywhere now.